### PR TITLE
Workaround MSYS2/chocolatey issue again

### PR DESCRIPTION
**Pull request by mati865**

### CODE REVIEW

**Comment by mati865** [comment]
Installing some MSYS2 packages prints `0 [main] pacman 898 dofork: child -1 - forked process 3560 died unexpectedly, retry 0, exit code 0xC0000142, errno 11` but this should work for some time though...

r? @pietroalbini or @Mark-Simulacrum

File: src/ci/scripts/install-msys2-packages.sh [hunk 1]
```diff
@@ -6,8 +6,15 @@ IFS=$'\n\t'
...
-    # FIXME(mati865): hopefully temporary workaround for MSYS2 issue
-    pacman -Sy --noconfirm pacman
+    # FIXME(mati865): temporary workaround until chocolatey updates their MSYS2
+    base_url='https://ci-mirrors.rust-lang.org/rustc/msys2-repo/msys/x86_64'
+    curl ${base_url}/libzstd-1.4.4-2-x86_64.pkg.tar.xz -o libzstd-1.4.4-2-x86_64.pkg.tar.xz
+    curl ${base_url}/pacman-5.2.1-6-x86_64.pkg.tar.xz -o pacman-5.2.1-6-x86_64.pkg.tar.xz
+    curl ${base_url}/zstd-1.4.4-2-x86_64.pkg.tar.xz -o zstd-1.4.4-2-x86_64.pkg.tar.xz
+    pacman -U --noconfirm libzstd-1.4.4-2-x86_64.pkg.tar.xz pacman-5.2.1-6-x86_64.pkg.tar.xz \
+        zstd-1.4.4-2-x86_64.pkg.tar.xz
+    rm libzstd-1.4.4-2-x86_64.pkg.tar.xz pacman-5.2.1-6-x86_64.pkg.tar.xz zstd-1.4.4-2-x86_64.pkg.tar.xz
```
Comment (pietroalbini) [inline src/ci/scripts/install-msys2-packages.sh]: The line is too long, tidy failed.

Comment (pietroalbini) [reply src/ci/scripts/install-msys2-packages.sh]: Well you fixed it as I commented :D

**Comment by pietroalbini** [comment]
@bors delegate+

r=me once check CI passes

File: src/ci/scripts/install-msys2-packages.sh (hunk 1 above)
Comment (mati865) [reply src/ci/scripts/install-msys2-packages.sh]: Oof, pushed the fix the same second as you commented...

**Comment by mati865** [comment]
`mingw-check is green`

@bors r=pietroalbini p=9001

**Pull request merged**